subdaos/overview.md

```diff
@@ -1,11 +1,11 @@
 # SubDAOs
 
-SubDAOs are semi-independent, specialized DAOs that are formed by MakerDAO and designed for protocol alignment. There are different types of subDAOs, and each type has a different role. 
-
 {% hint style="warning" %}
 This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date.
 {% endhint %}
 
+SubDAOs are semi-independent, specialized DAOs that are formed by MakerDAO and designed for protocol alignment. There are different types of subDAOs, and each type has a different role. 
```

Comment:
I'd remove the "formed by MakerDAO" line. Maybe "SubDAOs are semi-independent DAOs that are linked to Maker Governance. They are designed to increase innovation and experimentation while limiting risk to Maker Core."